Inference:
    ros__parameters:
        model_name: "policy.onnx"
        act_alpha: 1.0
        gyro_alpha: 1.0
        angle_alpha: 1.0
        frame_stack: 10
        decimation: 5
        intra_threads: -1
        dt: 0.004
        vx: 0.3
        vy: 0.0
        dyaw: 0.0
        obs_scales_lin_vel: 1.0
        obs_scales_ang_vel: 0.25
        obs_scales_dof_pos: 1.0
        obs_scales_dof_vel: 0.05
        obs_scales_gravity_b: 1.0
        clip_observations: 18.0
        action_scale: 0.25
        clip_actions: 18.0
        usd2urdf: [0, 6, 12, 1, 7, 13, 18, 2, 8, 14, 19, 3, 9, 15, 20, 4, 10, 16, 21, 5, 11, 17, 22]
        joint_lower_limit: [-0.6,-0.15,-1.2,-0.5,-0.5,-0.15,-0.15,-0.8,-1.2,-0.5,-0.5,-0.15,0.0,-1.0,-0.15,-0.3,-1.00,0.0,-1.0,-0.5,-0.3,-1.00,0.0]
        joint_higher_limit: [0.15, 0.8, 0.8, 1.6, 0.7, 0.15, 0.6, 0.15, 0.8, 1.6, 0.7, 0.15,0.0, 0.7, 0.5, 0.3, 0.5, 0.0, 0.7, 0.15, 0.3, 0.5, 0.0]

